### PR TITLE
Additional information hyperlinks.md

### PR DESCRIPTION
**Pull request by ShiroNotFriday**

### CODE REVIEW

File: src/guides/hyperlinks.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
 A hyperlink is clickable-text. When the user clicks on the text, it directs them to a certain URL.
 
 ## General Hyperlinks
-You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md) and in [webhook](../guides/webhooks.md) content/description.
+You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md), [webhook](../guides/webhooks.md) content/description, slash command response content and ephemeral message content.
```
Comment (MineBartekSA) [inline src/guides/hyperlinks.md:5]: Add a link to the slash command guide.
Only slash command responses can be ephemeral, so `slash command response content` should be enough.

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

File: src/guides/hyperlinks.md (hunk 1 above)
Comment (ShiroNotFriday) [reply src/guides/hyperlinks.md:5]: An ephemeral can be called not only by the slash command, but also by interaction with a button, an option in a menu, and a modal. In general, just by an interaction (the slash command is also a kind of interaction, but without the `$onInteraction`).

Comment (ShiroNotFriday) [reply src/guides/hyperlinks.md]: ephemeral.md should be updated. I think I will do this along with updating slashCommands.md. 
I have it planned.

Comment (ShiroNotFriday) [reply src/guides/hyperlinks.md]: Oh

Comment (MineBartekSA) [reply src/guides/hyperlinks.md]: Ephemeral messages aren't a different type of response, they are just a normal interaction response with a flag.
That's why I don't feel the need to mention them here.

File: src/guides/hyperlinks.md [hunk 2]
```diff
@@ -2,14 +2,16 @@
 A hyperlink is clickable-text. When the user clicks on the text, it directs them to a certain URL.
 
 ## General Hyperlinks
-You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md) and in [webhook](../guides/webhooks.md) content/description.
+You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md), [webhook](../guides/webhooks.md) content/description, [slash command](../guides/slashCommands.md) response content and ephemeral [$onInteraction](../callbacks/onInteraction.md) response content.
```
Comment (MineBartekSA) [inline src/guides/hyperlinks.md:5]: Ok, like, no `ephemeral`. All `$onInteraction[]` response content allows for hyperlinks, not only ephemeral.
And two more minor things, you put `,` before `and`, and change `$onIntraction` to look like the other functions.

Comment (ShiroNotFriday) [reply src/guides/hyperlinks.md:5]: > Ok, like, no ephemeral. All $onInteraction[] response content allows for hyperlinks, not only ephemeral.

https://user-images.githubusercontent.com/70456337/178481618-8e9885a2-e6e4-4962-9718-c5da03dd5045.mp4

File: src/guides/hyperlinks.md [hunk 3]
```diff
@@ -2,14 +2,16 @@
...
-You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md) and in [webhook](../guides/webhooks.md) content/description.
+You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md), [webhook](../guides/webhooks.md) content/description, [slash command](../guides/slashCommands.md) response content, and ephemeral [`$onInteraction`](../callbacks/onInteraction.md) response content.
 
 ### Syntax
 ```
 [text\](link)
 ```
 > **Note:** This is the syntax for BDScript 2 and BDScript Unstable. For the BDScript, the syntax is `[text](link)`.
 
+> **Note:** In the case of using hyperlinks inside slash command response content and ephemeral message content, the syntax for BDScript should be used. Does not apply to hyperlinks that are inside functions that support hyperlinks.
```
Comment (MineBartekSA) [inline src/guides/hyperlinks.md:13]: should be `or ephemeral response content`

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

File: src/guides/hyperlinks.md [hunk 4]
```diff
@@ -2,14 +2,16 @@
 A hyperlink is clickable-text. When the user clicks on the text, it directs them to a certain URL.
 
 ## General Hyperlinks
-You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md) and in [webhook](../guides/webhooks.md) content/description.
+You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md), [webhook](../guides/webhooks.md) content/description, [slash command](../guides/slashCommands.md) response content or ephemeral [`$onInteraction`](../callbacks/onInteraction.md) response content.
```
Comment (MineBartekSA) [inline src/guides/hyperlinks.md:5]: Wrong line with the `or`. In this the `and` was perfect.

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

File: src/guides/hyperlinks.md (hunk 4 above)
Comment (ShiroNotFriday) [reply src/guides/hyperlinks.md:5]: Ouch, I overlooked.

**Review by MineBartekSA — APPROVED** [review]
Looks great.
I'll wait for Noit to review this too before merging

**Review by noituri — APPROVED** [review]

**Pull request merged**